pkg/api/latest/latest.go

```diff
@@ -30,8 +31,21 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 )
 
+func init() {
+	version := os.Getenv("KUBE_API_LATEST_VERSION")
```

Comment:
I recommend making this take a comma-separated list of versions instead of the switching behavior. The switching behavior is very inflexible.